lib/rspec/core/reporter.rb

```diff
@@ -231,6 +231,7 @@ def ensure_listeners_ready
       return if @setup
 
       @setup_default.call
+      setup_profiler
```

Comment:
Should we remove the other `setup_profiler` call site and just let this be the one call site for it, since, AFAIK, the other one is no longer necessary with this?  Also, can we make `setup_profiler` private?

Reply:
Inlined it into the setup call.